src/auxiliary.h

```diff
@@ -0,0 +1,23 @@
+#ifndef AUXILIARY_H_
+#define AUXILIARY_H_
+
+#include <julia.h>
+
+// Helper function to set JULIA_DEPOT_PATH environment variable
+void update_depot_path(const char * project_directory, const char * depot_path);
+
+// Function for more helpful error messages
+#define LOC __func__, __FILE__, __LINE__
+void print_and_die(const char* message, const char* func, const char* file, int lineno);
+
+// Function to determine debug level
+int show_debug_output();
+
+// Function to evaluate julia REPL string with exception handling
+jl_value_t* checked_eval_string(const char* code, const char* func, const char* file,
+                                int lineno);
+
+// Function to get and store function pointers from julia to c functions
```

Comment:
```suggestion
// Function to get and store function pointers from Julia to c functions
```
You also write "C" and not "c" for the language 😬

Reply:
That's right. I try to find and change other occurrences referring to the language.